liquibase-service/src/main/resources/db/rdb_modern/routines/023-sp_d_interview_postprocessing-001.sql

```diff
@@ -0,0 +1,614 @@
+        INSERT INTO dbo.D_INTERVIEW (
+        D_INTERVIEW_KEY,
+        IX_STATUS_CD,
+        IX_DATE,
+        IX_INTERVIEWEE_ROLE_CD,
+        IX_TYPE_CD,
+        IX_LOCATION_CD,
+        LOCAL_ID,
+        RECORD_STATUS_CD,
+        RECORD_STATUS_TIME,
+        ADD_TIME,
+        ADD_USER_ID,
+        LAST_CHG_TIME,
+        LAST_CHG_USER_ID,
+        VERSION_CTRL_NBR,
+        IX_STATUS,
+        IX_INTERVIEWEE_ROLE,
+        IX_TYPE,
+        IX_LOCATION
+        ' + CASE
+        WHEN @Col_number > 0 THEN ',' +
+        (SELECT STRING_AGG(QUOTENAME(RDB_COLUMN_NM), ',')
+        FROM dbo.nrt_metadata_columns) + ') '
+        ELSE ')' end +
+                          ' SELECT 
+                          li.D_INTERVIEW_KEY,
+                          ix.IX_STATUS_CD,
+                          ix.IX_DATE,
+                          ix.IX_INTERVIEWEE_ROLE_CD,
+                          ix.IX_TYPE_CD,
+                          ix.IX_LOCATION_CD,
+                          ix.LOCAL_ID,
+                          ix.RECORD_STATUS_CD,
+                          ix.RECORD_STATUS_TIME,
+                          ix.ADD_TIME,
+                          ix.ADD_USER_ID,
+                          ix.LAST_CHG_TIME,
+                          ix.LAST_CHG_USER_ID,
+                          ix.VERSION_CTRL_NBR,
+                          ix.IX_STATUS,
+                          ix.IX_INTERVIEWEE_ROLE,
+                          ix.IX_TYPE,
+                          ix.IX_LOCATION
+                          ' + CASE
+                          WHEN @Col_number > 0 THEN ',' +
+                          (SELECT STRING_AGG('pv.' + QUOTENAME(RDB_COLUMN_NM), ',')
+                          FROM dbo.nrt_metadata_columns)
+                          ELSE '' END +
+                          'FROM #L_INTERVIEW_N li
+                          LEFT JOIN #INTERVIEW_INIT ix
+                              ON li.interview_uid = ix.interview_uid'
+            + CASE
+            WHEN @Col_number > 0 THEN
+         ' LEFT JOIN (
+        SELECT interview_uid, ' + @PivotColumns + '
+    FROM (
+        SELECT 
+            interview_uid, 
+            rdb_column_nm, 
+            answer_val
+        FROM 
+            #INTERVIEW_ANSWERS
+    ) AS SourceData
+    PIVOT (
+        MAX(answer_val) 
+        FOR rdb_column_nm IN (' + @PivotColumns + ')
+    ) AS PivotTable) pv 
+    ON pv.interview_uid = ix.interview_uid'
+        ELSE '' END;
+
+
+        exec sp_executesql @Insert_sql;
+
+
+        SELECT @ROWCOUNT_NO = @@ROWCOUNT;
+
+        INSERT INTO [DBO].[JOB_FLOW_LOG]
+        (BATCH_ID, [DATAFLOW_NAME], [PACKAGE_NAME], [STATUS_TYPE], [STEP_NUMBER], [STEP_NAME], [ROW_COUNT])
+        VALUES (@BATCH_ID, 'D_INTERVIEW', 'D_INTERVIEW', 'START', @PROC_STEP_NO, @PROC_STEP_NAME, @ROWCOUNT_NO);
+
+        COMMIT TRANSACTION;
+
+        BEGIN TRANSACTION;
+
+        SET @PROC_STEP_NO = @PROC_STEP_NO + 1;
+        SET @PROC_STEP_NAME = 'UPDATE D_INTERVIEW';
```

Comment:
The update to D_Interview should go before the insert.

Reply:
Fixed, thank you!